src/main/java/org/ethereum/beacon/discovery/liveness/LivenessChecker.java

```diff
@@ -3,22 +3,32 @@
  */
 package org.ethereum.beacon.discovery.liveness;
 
+import java.time.Clock;
+import java.time.Duration;
 import java.util.HashSet;
 import java.util.LinkedHashSet;
 import java.util.Set;
 import java.util.concurrent.CompletableFuture;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.ethereum.beacon.discovery.database.ExpirationSet;
 import org.ethereum.beacon.discovery.schema.NodeRecord;
 
 public class LivenessChecker {
   private static final Logger LOG = LogManager.getLogger();
 
   static final int MAX_CONCURRENT_PINGS = 3;
   static final int MAX_QUEUE_SIZE = 1000;
+  static final int MAX_IGNORE_SET_SIZE = 1000;
+  static final Duration ignoreDuration = Duration.ofSeconds(30);
```

Comment:
nit: This is a constant so should be named `IGNORE_DURATION`.

Reply:
yeah, was careless